src/transport/TcpRemotingClient.cpp

```diff
@@ -482,7 +498,7 @@ void TcpRemotingClient::ProcessData(const MemoryBlock& mem, const string& addr)
     LOG_DEBUG("find_response opaque:%d", opaque);
     processResponseCommand(pRespondCmd, pFuture);
   } else {
-    processRequestCommand(pRespondCmd, addr);
```

Comment:
i suggest continue to use processRequestCommand, put m_handleService.post() to it, it will more clear to readers

Reply:
No. Do it will write another function for call processRequest and invokeOneway, it is unnecessary.